src/openMVG/dataio/FeedProvider.cpp

```diff
@@ -44,25 +45,38 @@ FeedProvider::FeedProvider(const std::string &feedPath, const std::string &calib
 #endif
     }
   }
-  else if(bf::is_directory(bf::path(feedPath)))
+  else if(bf::is_directory(bf::path(feedPath)) || bf::is_directory(bf::path(feedPath).parent_path()))
```

Comment:
maybe a comment to explain:
parent_path returns 
"/foo/bar" returns"/foo"
"/foo/bar/" returns "/foo/bar"

Anyway if the user just give the relative path as "bar"...